.github/workflows/cicd.yml

```diff
@@ -0,0 +1,22 @@
+name: CICD
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        node: ['10', '12', '14']
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install dependencies
+        run: npm ci
+      - name: Test
```

Comment:
can you please add the TS type checks and the linter as well?

Reply:
@gergelyke the main `test` function runs the typings. Did you want that separated into its own line item? If so we can remove it from the main `npm test` command.

Reply:
my bad - disregard please :)

Reply:
No worries 😄 I did add a job for linting to run after test so we are all good there